packages/create-mc-app/src/tasks/update-application-constants.ts

```diff
@@ -34,12 +32,15 @@ function replaceEntryPointUriPathInConstants(
     ],
     retainLines: true,
   });
-
   if (result?.code) {
-    const prettierConfig = rcfile('prettier', {
-      cwd: options.projectDirectoryPath,
-    });
-    const formattedData = prettier.format(result.code + os.EOL, prettierConfig);
+    const prettierConfig = prettier.resolveConfig.sync(
+      options.projectDirectoryPath
+    );
+
+    const formattedData = prettier.format(
+      result.code + os.EOL,
+      prettierConfig!
```

Comment:
the provided type is `Options | null` while the expected type is `Options | undefined` so I thought about using type assertion here.